### PR TITLE
sqlx: 0.6 -> 0.7

### PR DESCRIPTION
**Pull request by ip1981**

### CODE REVIEW

File: .sqlx/query-1cabd2f674da323da9e0da724d3bcfe5f968b31500e8c8cf97fe16814bc04164.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (hsluoyz) [inline .sqlx/query-1cabd2f674da323da9e0da724d3bcfe5f968b31500e8c8cf97fe16814bc04164.json:1]: Remove the `.sqlx` folder

File: sqlx-data.json [hunk 2]
```diff
@@ -1,285 +0,0 @@
-{
```
Comment (hsluoyz) [inline sqlx-data.json:1]: Why remove it?

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@ip1981 fix all errors:

image

File: sqlx-data.json (hunk 2 above)
Comment (ip1981) [reply sqlx-data.json:1]: Please read sqlx changelog )

**Comment by ip1981** [comment]
There are no errors I can fix )

**Comment by hsluoyz** [comment]
@ip1981 plz fix: https://github.com/casbin-rs/sqlx-adapter/actions/runs/7013819881/job/19080557423?pr=80

image

**Comment by hsluoyz** [comment]
Replaced by: https://github.com/casbin-rs/sqlx-adapter/pull/87

**Pull request closed without merging**